src/Http/CookieFactory.php

```diff
@@ -77,7 +77,7 @@ public function __construct(Config $config)
      * @param  int     $maxAge
      * @return \Dflydev\FigCookies\SetCookie
      */
-    public function make($name, $value = null, $maxAge = null)
+    public function make(string $name, string $value, int $maxAge = null): SetCookie
     {
         $cookie = SetCookie::create($this->getName($name), $value);
```

Comment:
A value is now required here (necessary for the old library). Is that purposeful, or should we make this null default?

Reply:
I made it default, because as you said a cookie without value makes no sense. Also the underlying library will simply cast value to string (and then urlencode it)...

Reply:
I went back with nullable because it made the code cleaner.